containers/python/caffe2_container_entry.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+IMPORT_ERROR_RETURN_CODE=3
+
+echo "Attempting to run Caffe2 container without installing dependencies"
```

Comment:
Change this to "Attempting to run Caffe2 ONNX container without installing dependencies"